layers.json

```diff
@@ -2279,6 +2279,10 @@
         "af-south-1": 3,
         "me-south-1": 3
     },
+    "snmp-php-81": {
+        "us-east-1": 4,
+        "us-east-2": 4
+    },
```

Comment:
This is not needed. We will update this file automatically when releasing new versions

```suggestion
```